README.md

```diff
@@ -0,0 +1,95 @@
+# FarmLink
+Our capstone project is a web application that helps FarmLink, a non-profit organization that moves food surplus from farms around the country to food banks in need, automate the process of matching a food bank with a farm.
+# How to get application running!
+In this README file we will briefly explain how to get this application up and running on your computer. Feel free to reach out to the contributors to the repo for any further assistance.
+## Step #1: Clone the Repository
+Navigate to the directory where you would like to clone the repository. Then navigate to the main page of the repository, press the green code button, and then copy the link under "Clone with HTTPS".
+
+Now enter the following command into terminal:
+
+    git clone https://hostname/YOUR_USERNAME/YOUR-REPOSITORY
+At this point all the folders within the repository should have successfully downloaded locally.
+## Step #2: Download Dependencies 
+If you haven't installed npm, navigate to this [download tutorial](https://www.npmjs.com/get-npm) and do so before continuing.
+
+Now navigate to the view directory within the code base an run the following command:
+
+    ~/farmlink/view npm install
```

Comment:
This assumes that the user cloned into `~/farmlink`, which is probably not the case.  You might want to mention this assumption explicitly.